grails-app/views/contact.gsp

```diff
@@ -0,0 +1,205 @@
+                        </button>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-6 col-md-offset-3 text-center">
+                        <h3><small>Or drop us a note</small></h3>
+                        <form class="contact">
+                            <div class="row">
+                                <div class="col-md-6">
+                                    <input type="text" class="form-control" placeholder="First Name">
+
+                                </div>
+                                <div class="col-md-6">
+                                    <input type="text" class="form-control" placeholder="Last Name">
+
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-md-6">
+                                    <input type="text" class="form-control" placeholder="Email">
+
+                                </div>
+                                <div class="col-md-6">
+                                    <input type="text" class="form-control" placeholder="Subject">
+
+                                </div>
+                            </div>
+
+                            <textarea class="form-control" placeholder="Message" rows="7" ></textarea>
+
+                            <div class="row">
+                                <div class="col-md-6 col-md-offset-3">
+                                    <button class="btn btn-primary btn-fill btn-block">Send </button>
+                                </div>
+                            </div>
+
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<footer class="footer-demo">
+    <div class="container">
+        <nav class="pull-left">
+
+            <ul>
+                <li><a href="https://www.Groovy school.com">Groovy School</a></li>
+                <li><a href="http://blog.creative-tim.com"> About</a></li>
+                <li><a href="https://www.creative-tim.com">Catalog</a></li>
+            </ul>
+        </nav>
+
+        <div class="copyright pull-right">
+            &copy; 2019 - GR8DI
+        </div>
+    </div>
+</footer>
+</body>
+
+<script src="../assets/js/jquery-1.10.2.js" type="text/javascript"></script>
+<script src="../assets/js/jquery-ui-1.10.4.custom.min.js" type="text/javascript"></script>
+
+<script src="../bootstrap3/js/bootstrap.js" type="text/javascript"></script>
+
+<!--  Plugins -->
+<script src="../assets/js/ct-paper-checkbox.js"></script>
+<script src="../assets/js/ct-paper-radio.js"></script>
+<script src="../assets/js/bootstrap-select.js"></script>
+<script src="../assets/js/bootstrap-datepicker.js"></script>
+<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?v=3.exp"></script>
+
+<script src="../assets/js/ct-paper.js"></script>
+
+<script>
```

Comment:
we don't need any facebook pixel code. we won't be running any facebook adds in the nearest future. Please, could you remove this script section?